app/src/main/java/com/github/se/gatherspot/firebase/EventFirebaseConnection.kt

```diff
@@ -280,6 +280,27 @@ class EventFirebaseConnection : FirebaseConnectionInterface<Event> {
     return eventsFromQuerySnapshot(querySnapshot)
   }
 
+  suspend fun fetchAllInPerimeter(
+      latitude: Double,
+      longitude: Double,
+      degrees: Double
+  ): MutableList<Event> {
+
+    val querySnapshot: QuerySnapshot =
```

Comment:
this is a great implementation to not fetch all events but only the events nearby